### DISCUSSION TITLE
Graph labels/overlays to show data over left/right Y axis

### DISCUSSION THREAD

**Discussion by RichiH** [grafana/grafana#38278] [Dashboards, Panels & Visualizations]
See the overlay on the right-hand Y axis:

![image](https://user-images.githubusercontent.com/754723/117981108-f3823480-b334-11eb-8996-fd2d7dae52ca.png)

And this tech demo by @torkelo :

https://user-images.githubusercontent.com/754723/117981280-23c9d300-b335-11eb-9a3e-52c361b108f4.mp4

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
